src/language-markdown/index.js

```diff
@@ -9,21 +9,13 @@ const languages = [
     override: {
       since: "1.8.0",
       parsers: ["remark"],
-      vscodeLanguageIds: ["markdown"],
-      extensions: [
-        ".md",
-        ".markdown",
-        ".mdown",
-        ".mdwn",
-        ".mkd",
-        ".mkdn",
-        ".mkdown",
-        ".ronn",
-        ".workbook"
-      ]
+      vscodeLanguageIds: ["markdown"]
     },
     extend: {
       filenames: ["README"]
+    },
+    exclude: {
```

Comment:
`createLanguage` is not support `exclude`, we need implement this, please look in code, before adding somethings